dockstore-webservice/src/main/resources/migrations.1.5.0.xml

```diff
@@ -165,4 +165,18 @@
             <column name="dbupdatedate" type="timestamp"/>
         </addColumn>
     </changeSet>
+    <changeSet author="gluu" id="migrate-legacy-verification-information">
+        <comment>Migrate the verified workflow versions into the new sourcefile_verified table</comment>
+        <sql dbms="postgresql">
```

Comment:
Close enough, but note that the comment can be put directly on the sql tag 
https://www.liquibase.org/documentation/changes/sql.html